virttest/ceph.py

```diff
@@ -207,3 +221,36 @@ def create_config_file(ceph_monitor):
         with open(ceph_cfg, 'w+') as f:
             f.write('mon_host = %s' % ceph_monitor)
     return ceph_cfg
+
+
+def create_keyringfile(keyringfile, keyring=None, entityname=None,
+                       mode=None, caps=None):
```

Comment:
Since one keyring file can store zero (not sure if it is meaningful) or more keys, the interface, from my point of view, would be:
```python
def create_keyring(path, keys_desc=None, force=False)
    """
    :params path: The path the specified keyring file
    :params keys_desc: The description of each key
    :params force: Whether overwrite existed keyring file
    """
```
where the structure of `keys_desc` would be:
```python
[
    {
        'entity_name': 'client.admin',
        'key_content': ...,
        'capabilities': ...,
    },
    {
        ...
    }
    ...
]
```

Reply:
We made a decision that let the user prepare a keyring file, whose path will be passed to avocado-vt. So removed create and remove keyring functions.